### DISCUSSION TITLE
2.9 递归对象

### DISCUSSION THREAD

**Discussion by giscus** [csfive/composing-programs-zh#114] [Announcements]
# 2.9 递归对象

SICP Python 中文翻译

https://composingprograms.netlify.app/2/9

**Comment by bozagina** [comment]
求解答，partitions部分的代码似乎有问题？
TypeError: can only concatenate tuple (not "Link") to tuple

**Reply by ShihchiaYJMd** [reply]
需要__radd__=__add__保证左右加都可以，书里漏了一行代码
